btr-web/btr-main-app/components/person-info/Control.vue

```diff
@@ -0,0 +1,374 @@
+      >
+        <!-- eslint-disable-next-line -->
+        <template #default="{ item, open }">
+          <UButton :padded="false" variant="ghost">
+            <template #leading>
+              <UIcon
+                name="i-heroicons-chevron-down-20-solid"
+                class="transform transition-transform duration-200"
+                :class="[open && 'rotate-180']"
+              />
+            </template>
+            {{ item.label }}
+          </UButton>
+        </template>
+        <!-- eslint-disable-next-line -->
+        <template #item="{ item }">
+          <!-- eslint-disable-next-line -->
+          <span v-html="item.content" />
+        </template>
+      </UAccordion>
+    </div>
+
+    <div v-if="item.controlOther" class="mt-3">
+      <span class="font-bold italic">
+        {{ $t('labels.other') }}
+      </span>
+      <p>{{ item.controlOther }}</p>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { SiSchemaType, SiControlOfDirectorsSchemaType } from '~/utils/si-schema/definitions'
+import SiSchemaToBtrBodsConverters from '~/utils/btr-bods/si-schema-to-btr-bods-converters'
+import { PercentageRangeE } from '~/enums/percentage-range-e'
+const t = useNuxtApp().$i18n.t
+
+type IconType = { src: string; alt: string }
+
+const prop = defineProps<{ item: SiSchemaType }>()
+const interests = SiSchemaToBtrBodsConverters.getInterests(prop.item)
+
+const INTERESTS_SHARES_JOINT = 'controlType.shares.actingJointly'
+const INTERESTS_SHARES_CONCERT = 'controlType.shares.inConcertControl'
+const INTERESTS_VOTES_JOINT = 'controlType.votes.actingJointly'
+const INTERESTS_VOTES_CONCERT = 'controlType.votes.inConcertControl'
+const INTERESTS_DIRECTORS_JOINT = 'controlType.directors.actingJointly'
+const INTERESTS_DIRECTORS_CONCERT = 'controlType.directors.inConcertControl'
+
+const sharesJointly = computed(() => {
+  return interests.find(i => i.details === INTERESTS_SHARES_JOINT)?.connectedIndividuals || []
+})
+
+const sharesInConcert = computed(() => {
+  return interests.find(i => i.details === INTERESTS_SHARES_CONCERT)?.connectedIndividuals || []
+})
+
+const votesJointly = computed(() => {
+  return interests.find(i => i.details === INTERESTS_VOTES_JOINT)?.connectedIndividuals || []
+})
+
+const votesInConcert = computed(() => {
+  return interests.find(i => i.details === INTERESTS_VOTES_CONCERT)?.connectedIndividuals || []
+})
+
+const directorsJointly = computed(() => {
+  return interests.find(i => i.details === INTERESTS_DIRECTORS_JOINT)?.connectedIndividuals || []
+})
+
+const directorsInConcert = computed(() => {
+  return interests.find(i => i.details === INTERESTS_DIRECTORS_CONCERT)?.connectedIndividuals || []
+})
+
+const sharesAccordionItems = computed(() => {
+  const rv = []
+  const jointly = sharesJointly
+  const inconcert = sharesInConcert
+  if (jointly.value.length > 0) {
+    rv.push({
+      label: `Acting jointly (${jointly.value.length + 1})`,
```

Comment:
Thanks for the updates! One last thing here and below, the '+1' should be removed so the counts are correct